chispa/schema_comparer.py

```diff
@@ -95,11 +103,12 @@
         if not ignore_metadata and sf1.metadata != sf2.metadata:
             return False
         else:
-            return are_datatypes_equal_ignore_nullable(sf1.dataType, sf2.dataType)
+            return are_datatypes_equal_ignore_nullable(sf1.dataType, sf2.dataType)  # type: ignore[no-any-return, no-untyped-call]
 
 
 # deprecate this
```

Comment:
Let's, maybe, add a deprecation warning?

Reply:
There's multiple functions with `# deprecate this` comment. Let's leave solving that for a follow-up PR.